burst-kotlin-plugin/src/main/kotlin/app/cash/burst/kotlin/BurstApis.kt

```diff
@@ -37,6 +37,10 @@ internal class BurstApis private constructor(
         return BurstApis(pluginContext, junitPackage)
       }
 
+      if (pluginContext.referenceClass(junit5TestClassId) != null) {
+        return BurstApis(pluginContext, junit5Package)
+      }
```

Comment:
We might need to change strategies here in a follow-up, as these can be mixed together with JUnit 4.

Reply:
Oooh good catch. Test case coming.